.github/workflows/update-current-version.yml

```diff
@@ -0,0 +1,27 @@
+# This workflow will update the readme file to point to the current version of Data Profiler.
+
+name: Update Current Version Tag
+
+on:
+  push:
+    branches:
+      - 'main'
+
+jobs:
+  update-readme:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v4
+      with:
+        fetch-depth: 0
+
+    - name: Update README.md
+    
+      run: |
+        latest_tag=$(git describe --tags --abbrev=0)
+        sed "s|<workflow_to_replace_with_live_version_tag>|$latest_tag|g" README.md > tmp.md && mv tmp.md README.md
+        git config --local user.email "mishalksa@hotmail.com"
+        git config --local user.name "GitHub Action"
+        git commit -am "Update README with latest version tag"
+        git push
```

Comment:
hmm -- nifty what you did here but this creates a situation where each commit that fixes this is tagged to your account, which potentially isn't very robust long-term

Reply:
not something that is production-ready per se

Reply:
I could create a dummy account and sign the cla. Do you have any ideas on how to make it more robust?